lib/mix/phoenix.ex

```diff
@@ -88,12 +91,13 @@ defmodule Mix.Phoenix do
   Parses the attrs as received by generators.
   """
   def attrs(attrs) do
-    Enum.map attrs, fn attr ->
+    attrs
+    |> Enum.map fn attr ->
       case String.split(attr, ":", parts: 3) do
         [key, comp, value] -> {String.to_atom(key), {String.to_atom(comp), String.to_atom(value)}}
         [key, value]       -> {String.to_atom(key), String.to_atom(value)}
         [key]              -> {String.to_atom(key), :string}
-      end
+      end |> validate_attr
```

Comment:
Lets rework this pipeline with parens. Enum.map should have parens and the piping precedence should be obvious